src/components/event-delegation.js

```diff
@@ -74,6 +74,11 @@ function attachBubbleEventListeners(doc) {
             var curNode = event.target;
             if (!curNode) {
                 return;
+            } else if (typeof SVGElementInstance !== 'undefined' && curNode instanceof SVGElementInstance) {
+                // event.target of an SVGElementInstance does not have a
+                // `getAttribute` function in IE 11.
+                // See https://github.com/marko-js/marko/issues/796
+                curNode = curNode.correspondingUseElement || curNode;
```

Comment:
Wouldn't it be better to drop the else block and just always run the following line of code?:

```js
curNode = curNode.correspondingUseElement || curNode;
```
Since `correspondingUseElement` will not be a property on normal HTML elements I don't see it causing a problem.